### ISSUE TITLE
eService lists - remove "Client" from list

### ISSUE THREAD

**Issue by JasonSchneider19** [Sanjel-Energy-Services/eService_WorkItems#278]
@awangatsanjel 
remove "Client" from Plug Loading Head and Tubular Plugs
![image](https://github.com/user-attachments/assets/c5356ca3-5457-483b-905e-bd49f238f970)
![image](https://github.com/user-attachments/assets/86692624-23ea-4d56-a5fa-490797f3b5b7)
This will leave  "Third Party" and "Sanjel" in the list

**Comment by awangatsanjel** [comment]
@JasonSchneider19 
Do we want to keep Client in DownHoleTools section?

**Comment by awangatsanjel** [comment]
@JasonSchneider19 Can you clarify this question?

**Comment by JasonSchneider19** [comment]
@awangatsanjel 
Yes, please remove Client from the DownHoleTools section as well. make all the same
